Gems/StartingPointInput/Code/Source/InputHandlerNodeable.h

```diff
@@ -28,7 +28,6 @@ namespace StartingPointInput
     public:
         InputHandlerNodeable() = default;
         virtual ~InputHandlerNodeable();
-        InputHandlerNodeable(const InputHandlerNodeable&) = default;
         InputHandlerNodeable& operator=(const InputHandlerNodeable&) = default;
```

Comment:
good catch, btw do we really need assignment operator for nodeable?
same for copy constructor, do we really use copy constructor in any use case?